src/cli.ts

```diff
@@ -192,9 +192,9 @@ function transformMarkdown(buf: Buffer) {
   let state = 'root'
   let codeBlockEnd = ''
   let prevLineIsEmpty = true
-  const jsCodeBlock = /^(```+|~~~+)(js|javascript)$/
-  const shCodeBlock = /^(```+|~~~+)(sh|bash)$/
-  const otherCodeBlock = /^(```+|~~~+)(.*)$/
+  const jsCodeBlock = /^(```+|~~~+)(?:js|javascript)$/
+  const shCodeBlock = /^(```+|~~~+)(?:sh|bash)$/
```

Comment:
thought it made sense to use a none capturing group here given we aren't using the value